doc/cephfs/client-auth.rst

```diff
@@ -2,15 +2,15 @@
 CephFS Client Capabilities
 ================================
 
-Use Ceph authentication capabilities to restrict your file system clients
-to the lowest possible level of authority needed.
+Ceph authentication capabilities are used to restrict file system clients to
+the lowest level of authority necessary.
 
-.. note:: Path restriction and layout modification restriction are new features
-    in the Jewel release of Ceph.
+.. note:: Path restriction and layout-modification restriction were introduced
+   in the Jewel release of Ceph.
```

Comment:
Maybe we don’t care about Infernalis and earlier?  Remove?

Reply:
I'm on the fence about removing stuff like this.

The case for removing information about when features were introduced: this is a technical manual, meant to explain how to use Ceph. This is not a historical record of the development of Ceph.

The case against removing information about when features were introduced: It's nice to know when features were introduced, and gives a sense of Ceph as an ongoing concern that responds to the world in which it serves the ends of its users and developers.